src/webgpu/shader/execution/expression/call/builtin/degrees.spec.ts

```diff
@@ -11,38 +11,32 @@ import { makeTestGroup } from '../../../../../../common/framework/test_group.js'
 import { GPUTest } from '../../../../../gpu_test.js';
 import { TypeAbstractFloat, TypeF16, TypeF32 } from '../../../../../util/conversion.js';
 import { FP } from '../../../../../util/floating_point.js';
-import { fullF16Range, fullF32Range, fullF64Range } from '../../../../../util/math.js';
 import { makeCaseCache } from '../../case_cache.js';
 import { allInputSources, onlyConstInputSource, run } from '../../expression.js';
 
 import { abstractBuiltin, builtin } from './builtin.js';
 
 export const g = makeTestGroup(GPUTest);
 
+const cases = (['f32', 'f16'] as const)
```

Comment:
Maybe
``` suggestion
const f32_f16_cases = (['f32', 'f16'] as const)
```

Reply:
Merged the abstract generator in, generating an empty array of Cases when non_const.